src/Anonymization/Anonymizer/Options.php

```diff
@@ -51,4 +55,75 @@ public function toDisplayString(): string
             \array_values($this->options),
         ));
     }
+
+    public function getString(string $name, string $default = null, bool $required = false): ?string
+    {
+        $value = $this->get($name, $default, $required);
+        if ($value === null || is_scalar($value) || $value instanceof \Stringable) {
+            return (string) $value;
+        }
+
+        throw new \InvalidArgumentException(\sprintf("Option '%s' value should be a string", $name));
+    }
+
+    public function getInt(string $name, int $default = null, bool $required = false): ?int
+    {
+        $value = $this->get($name, $default, $required);
+
+        if (\is_numeric($value)) {
+            return (int) $value;
+        }
+
+        throw new \InvalidArgumentException(\sprintf("Option '%s' value should be an int", $name));
+    }
+
+    public function getFloat(string $name, float $default = null, bool $required = false): ?float
+    {
+        $value = $this->get($name, $default, $required);
+
+        if (\is_numeric($value)) {
+            return (float) $value;
+        }
+
+        throw new \InvalidArgumentException(\sprintf("Option '%s' value should be a float", $name));
+
+    }
+
+    public function getDate(string $name, \DateTimeImmutable $default = null, bool $required = false): ?\DateTimeImmutable
+    {
+        $value = $this->get($name, $default, $required);
+
+        if ($value instanceof \DateTimeInterface) {
```

Comment:
Aren't our configuration options not always strings ? Anyway better be safe than sorry I'm not against keeping this check.

Reply:
I try to anticipate that in future, one could configure anonymization with some PHP.

Also, we do not validate that the array given in `Options` class constructor is string only, so I added this check just in case :shrug: